scripts/badger/process_bribes_graviaura.py

```diff
@@ -133,16 +133,18 @@ def sell_weth():
         #  1. Swap WETH for AURA and deposit to GRAVI_AURA
         #  2. Swap WETH directly for GRAVI_AURA
         buy_amount_aura = int(
-            SAFE.cow.get_fee_and_quote(WETH, AURA, aura_share)["buyAmountAfterFee"]
+            SAFE.cow.get_fee_and_quote(WETH, AURA, aura_share, SAFE.address)["quote"][
+                "feeAmount"
```

Comment:
should not be instead: 

```suggestion
                "buyAmount"
```